wazo_amid_client/commands/config.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2022 The Wazo Authors  (see the AUTHORS file)
```

Comment:
Wrong copyright date. Make sure you have pre-commit installed

Reply:
Fixed in https://github.com/wazo-platform/wazo-amid-client/pull/18